tests/cli/test_cmds_mark_for_deployment.py

```diff
@@ -95,41 +94,6 @@ def test_mark_for_deployment_sad(mock_create_remote_refs, mock__log_audit, mock_
     assert not mock__log_audit.called
 
 
-@patch('paasta_tools.cli.cmds.mark_for_deployment.validate_service_name', autospec=True)
-@patch('paasta_tools.cli.cmds.mark_for_deployment.mark_for_deployment', autospec=True)
-@patch('paasta_tools.cli.cmds.mark_for_deployment.wait_for_deployment', autospec=True)
-@patch('paasta_tools.cli.cmds.mark_for_deployment.get_currently_deployed_sha', autospec=True)
-def test_paasta_mark_for_deployment_with_good_rollback(
-    mock_get_currently_deployed_sha,
-    mock_wait_for_deployment,
-    mock_mark_for_deployment,
-    mock_validate_service_name,
-):
-    class fake_args_rollback(fake_args):
-        auto_rollback = True
-        block = True
-        timeout = 600
-
-    mock_mark_for_deployment.return_value = 0
-    mock_wait_for_deployment.side_effect = TimeoutError
-    mock_get_currently_deployed_sha.return_value = "old-sha"
-    with patch('time.sleep', autospec=True):
-        assert mark_for_deployment.paasta_mark_for_deployment(fake_args_rollback) == 1
-    mock_mark_for_deployment.assert_any_call(
-        service='test_service',
-        deploy_group='test_deploy_group',
-        commit='d670460b4b4aece5915caf5c68d12f560a9fe3e4',
-        git_url='git://false.repo/services/test_services',
-    )
-    mock_mark_for_deployment.assert_any_call(
-        service='test_service',
-        deploy_group='test_deploy_group',
-        commit='old-sha',
-        git_url='git://false.repo/services/test_services',
-    )
-    mock_mark_for_deployment.call_count = 2
```

Comment:
lol, was this supposed to be an assertion of equality?

Reply:
Hehe. We decided we didn't need this test anymore.